packages/jira-adapter/src/filters/layouts/layout_service_operations.ts

```diff
@@ -101,14 +104,25 @@ export const getLayoutResponse = async ({
 }
 
 const fromlayoutConfigRespTolayoutConfig = (
-  containers: containerIssueLayoutResponse[]
+  layoutConfig: IssueLayoutConfiguration
 ): issueLayoutConfig => {
+  const { containers } = layoutConfig.issueLayoutResult
+  const fieldItemIdToMetaData = Object.fromEntries(layoutConfig.metadata.configuration.items.nodes
+    .filter(node => !_.isEmpty(node))
+    .map(node => {
+      const { fieldItemId } = node
+      delete node.fieldItemId
+      return [fieldItemId, node]
```

Comment:
I think that it's better to not delete a value in-place and prefer to create a copy of it.
```suggestion
      return [fieldItemId, _.omit(node, fieldItemId)]
```